sfepy/discrete/common/domain.py

```diff
@@ -216,6 +264,11 @@ def create_region(self, name, select, kind='cell', parent=None,
         region = visit_stack(stack, region_op,
                              region_leaf(self, self.regions, select,
                                          functions, tdim))
+
+        if extra_options is not None and 'mesh_dim' in extra_options:
```

Comment:
Maybe use () to make the condition clearer? (`if (extra_options is not None) and ('mesh_dim' in extra_options):`)